wiki/Beginner/en.md

```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
+One of the most important factors when playing *osu!* is your timing, much like other rhythm games! In this sense, if you've listened to a lot of music, it should be fairly easy to pick up on. The goal of accuracy and timing is to tap the notes at the correct time as precisely as possible.
 
 Each object in the game has an approach circle; a large ring that slowly approaches the hit circle itself. When the ring reaches the circle, it's time to tap it! If you pay attention to them carefully, it will help you recognize whether some notes are on half-beats or full beats, or whether you need to wait for the next note.
 
 Music works in beats and a rhythm. When you start a map (or even before you play) pay attention to the how the song sounds, and you can get a pretty good idea on the melody that you will be hitting circles to. Getting along with the music more closely will help you out in achieving better timing and coordination over it.
 
 ### Aiming
 
-In a point-and-click game like osu!, aiming is also another important factor. Being able to get the notes at the right time is pretty pointless without being able to get your cursor to the right location. In this case, the most effective way to improve in this sector is to simply play the game. It also helps to play harder and harder maps, since the spacing between notes will likely get bigger, making you move around more and tap the notes from wider distances.
+In a point-and-click game like *osu!*, aiming is also another important factor. Being able to get the notes at the right time is pretty pointless without being able to get your cursor to the right location. In this case, the most effective way to improve in this sector is to simply play the game. It also helps to play harder and harder maps, since the spacing between notes will likely get bigger, making you move around more and tap the notes from wider distances.
 
-The best way you can get practice for aiming without osu! is using your computer with the input device. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
+The best way you can get practice for aiming without *osu!* is using your computer with the input device. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
 
 ### Speed
 
 Eventually, when you get to harder difficulties, you will need to click the notes more rapidly. Improving in this fact is actually quite simple; all you need to do is push yourself into clicking or tapping the keyboard faster. You can play games that require you to hit input as fast as possible, like Cookie Clicker or other button pushing games (they are very easy to find online if you just Google it).
```

Comment:
"Improving in this is actually quite simple..." (Get rid of "this")
"You can play games that require you to input as fast as possible..." (Get rid of "hit")